app/assets/stylesheets/articles.scss

```diff
@@ -0,0 +1,28 @@
+  .card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    padding: 20px;
+    word-wrap: break-word;
+    background-color: #fff;
```

Comment:
define color constants to ensure if you're going to use a different color in the future there will be less effort.